geologist/child-theme.json

```diff
@@ -244,14 +244,14 @@
 					"background": "var(--wp--custom--color--tertiary)"
 				},
 				"typography": {
-					"fontSize": "20px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
 					"fontWeight": "400",
 					"lineHeight": "1.6"
 				}
 			},
 			"core/navigation": {
 				"typography": {
-					"fontSize": "18px"
+					"fontSize": "var(--wp--preset--font-size--normal)"
```

Comment:
Should this one be small?

Reply:
Quadrat is also normal. Same as the above comments.